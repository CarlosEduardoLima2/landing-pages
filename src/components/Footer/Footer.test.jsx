import { screen } from '@testing-library/react';
import { Footer } from '.';
import { renderTheme } from '../../styles/render-theme';

describe('<Footer />', () => {
  it('should render Footer', () => {
    const { container } = renderTheme(<Footer footerHtml="<h1>oi</h1>" />);
    expect(screen.getByRole('heading', { name: 'oi' })).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
