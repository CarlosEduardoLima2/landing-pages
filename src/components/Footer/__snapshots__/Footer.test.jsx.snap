// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render Footer 1`] = `
.c1 {
  text-align: center;
}

.c1 a {
  color: inherit;
  text-decoration: none;
  font-size: 1.6rem;
}

.c2 {
  font-size: 2.4rem;
}

.c0 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
}

<div>
  <div
    class="c0"
  >
    <footer
      class="c1"
    >
      <div
        class="c2"
      >
        <h1>
          oi
        </h1>
      </div>
    </footer>
  </div>
</div>
`;
